MAUI/DataForm/Validation.md

```diff
@@ -0,0 +1,306 @@
+    public string Password { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Customize error label text style
+
+The error label style can be customized by changing the [ErrorLabelTextStyle] property of [SfDataForm].
+
+{% tabs %}
+{% highlight xaml %}
+
+    <dataForm:SfDataForm
+        x:Name="dataForm">
+        <dataForm:SfDataForm.ErrorLabelTextStyle>
+            <dataForm:DataFormTextStyle FontSize="10" FontAttributes="Italic" TextColor="Violet" FontFamily="Roboto"/>
+        </dataForm:SfDataForm.ErrorLabelTextStyle>
+    </dataForm:SfDataForm>
+
+{% endhighlight %}
+{% endtabs %}
+
+You can also customize error label style for each editor using [ErrorLabelTextStyle] property of [DataFormItem]
+
+{% tabs %}
+{% highlight c# %}
+
+this.dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+ private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+{
+        if (e.DataFormItem != null)
+        {
+            if (e.DataFormItem.FieldName == "FirstName")
+            {
+                e.DataFormItem.ErrorLabelTextStyle = new DataFormTextStyle
+                {
+                    TextColor = Colors.DarkSeaGreen,
+                    FontSize = 8,
+                    FontAttributes = FontAttributes.Italic,
+                    FontFamily = "Roboto",
+                };
+            }
+        }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Customize valid message label text style
+
+The valid message label style can be customized by changing the [ValidMessageLabelTextStyle] property of [SfDataForm].
+
+{% tabs %}
+{% highlight xaml %}
+
+        <dataForm:SfDataForm
+        x:Name="dataForm">
+        <dataForm:SfDataForm.ValidMessageLabelTextStyle>
+            <dataForm:DataFormTextStyle FontSize="10" FontAttributes="Italic" TextColor="Violet" FontFamily="Roboto"/>
+        </dataForm:SfDataForm.ValidMessageLabelTextStyle>
+    </dataForm:SfDataForm>
+
+{% endhighlight %}
+{% endtabs %}
+
+You can also customize valid message label style for each editor using [ValidMessageLabelTextStyle] property of [DataFormItem]
+
+{% tabs %}
+{% highlight c# %}
+
+this.dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+ private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+{
+        if (e.DataFormItem != null)
+        {
+            if (e.DataFormItem.FieldName == "FirstName")
+            {
+                e.DataFormItem.ValidMessageLabelTextStyle = new DataFormTextStyle
+                {
+                    TextColor = Colors.DarkSeaGreen,
+                    FontSize = 8,
+                    FontAttributes = FontAttributes.Italic,
+                    FontFamily = "Roboto",
+                };
+            }
+        }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## Events
+
+#### ValidateForm
```

Comment:
## Validate the data form

You can get the validation details of the all the editors of data form using the [ValidateForm] event of the data form.

Note: This event will be raised once after the manual validation call using [SfDataForm.Validate()] method

## Validate the specific editor 

Allows you to validate specific editor in the data form.

Reply:
content changed